src/components/DonationHistory.jsx

```diff
@@ -61,7 +61,7 @@ class DonationHistory extends Component {
       historyItem.parentIds,
       historyItem,
     );
-    if (parents.length > 0) {
+    if (!historyItem.homeTxHash && parents.length > 0) {
```

Comment:
If donation has `homeTxHash` we guess it is direct donation, so we should not check donation's parent